vipps-recurring-api.md

```diff
@@ -1085,13 +1082,23 @@ The following properties are available for updating:
   "pricing": {
     "amount": 25000,
     "suggestedMaxAmount": 300000
+  },
+  "interval": {
+    "period": {
+        "count": 1,
+        "unit": "MONTH"
+    }
   }
 }
 ```
 
 Updating `amount` is only possible for agreements with `pricing.type:LEGACY`
+
 Updating `suggestedMaxAmount` is only possible for agreements with `pricing.type:VARIABLE`
 
+In order to update the interval from a RECURRING type to a FLEXIBLE type, you need to put an empty `interval` 
```

Comment:
Remove this text, we can put an example of later when we enable Flexible frequency for recurring